operations/workplace/people/working-at-mattermost/internal-transfers.md

```diff
@@ -1,6 +1,15 @@
-# Internal Transfers
+# Transfers & Promotions
 
-Internal transfers have an informal and formal component. Informally, a team member is encouraged to be curious and seek to learn about other roles, teams, projects, and Hiring Managers.
+We want staff to feel empowered to do the best work of their careers at Mattermost and enabled to drive the direction of their own growth trajectories at Mattermost. 
+
+## What's the difference between a transfer and a promotion?
+
+* Transfers occur when a team member moves to a new team or function and owns a different set of [AORs](https://handbook.mattermost.com/operations/operations/areas-of-responsibility). For example, a Customer Support Engineer would transfer to a Customer Success Engineer role, or vice versa.
```

Comment:
```suggestion
* Transfers occur when a team member moves to a new team or function and owns a different set of [AORs](https://handbook.mattermost.com/operations/operations/areas-of-responsibility). For example, a Customer Support Engineer transferring to a Product Manager role.
```

Reply:
I changed the transfer example so it's clear that transfers aren't only within a specific team or area of expertise. LMK if that's incorrect.

Reply:
Makes total sense, ty!